waku/v2/utils/wakuswitch.nim

```diff
@@ -0,0 +1,57 @@
+# Waku Switch utils.
+import
+  std/[options, tables, strutils, sequtils, os],
```

Comment:
Thanks for splitting this out! Are all of these imports necessary in the new module? Not sure, but I don't think e.g. `ping` and `enr` is used?
Imports can be messy, but good idea to try our best to not have unused/unnecessary imports.